styles/styles.css

```diff
@@ -0,0 +1,711 @@
+  font-weight: var(--ff-fontweight-black);
+  color: var(--word-clr);
+}
+
+.services-inner {
+  z-index: 2;
+  flex-basis: 1100px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 2em;
+}
+
+
+.service-box {
+  flex: 1 0 20em;
+  display: flex;
+  flex-direction: column;
+  border: 2px solid var(--primary-clr);
+  padding: 2em;
+  background-color: #FFF;
+  box-shadow:
+  0px 1.9px 40.6px -20px rgba(0, 0, 0, 0.01),
+  0px 5.1px 46.6px -20px rgba(0, 0, 0, 0.1),
+  0px 12.4px 35px -20px rgba(0, 0, 0, 0.4),
+  0px 5px 20px -20px rgba(0, 0, 0, 0.2);
+
+}
+
+/* -- TYPO SERVICE-BOX -- */
+
+.service-box h3 {
+  font-size: 1.25rem;
+  font-weight: var(--ff-fontweight-bold);
+  color: var(--primary-clr)
+}
+
+.service-box p {
+  font-size: 0.85rem;
+  font-weight: var(--ff-fontweight-light);
+  line-height: 1.2em;
+}
+
+.service-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  row-gap: 1em;  
+}
+
+.service__icon {
+  width:25%;
+}
+
+/* -- SECTION CTA FORM -- */
+
+.cta-outer {
+  padding: var(--padding-section) 0;
+  background-image: url(../assets/noise-texture.png), var(--gradient-clr);
+  clip-path: polygon(0 0, 100% calc(0px + 6vw), 100% 100%, 0 100%);
+}
+
+.cta-inner {
+  padding-top: 3em;
+  flex-basis: 768px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.4em;
+  color: #FFF;
+}
+
+.cta-content {
+  text-align: center;
+}
+
+.cta-content h2 {
+  font-size: var(--ff-fontsize-h1);
+  margin-bottom: 0.5em;
+}
+
+.cta-content p {
+font-family: var(--ff-fontfamily-roboto);
+font-size: 1.3rem;
+font-weight: var(--ff-fontweight-light);
+}
+
+form {
+  width: 100%;
+  display:flex;
+  flex-direction: column;
+  gap:2em;
+ }
+
+ .divider {
+  border: none;
+  width: 100%;
+ }
+
+input:not(.cta-btn), textarea {
+  width: 100%;
+  font-size: 0.86rem;
+  padding: 1em;
+  border-radius: 0.4em;
+  border: unset;
+ }
+
+input:focus, textarea:focus {
+  outline-color: var(--secondary-clr);
+}
+ 
+form .form-btn {
+   padding: 1.4em 6em;
+   align-self: center;
+ }
+
+/* -- SECTION FOOTER -- */
+
+footer {
+  background-color: var(--font-clr);
+  color: #FFF;
+  font-weight: var(--ff-fontweight-light);
+  font-size: 0.86rem;
+}
+
+.footer-inner {
+  display: flex;
+  justify-content: center;
+  padding: 2em 0;
+}
+
+/* -- MEDIA QUERIES MOBILE-- */
+
+@media only screen and (max-width: 480px)  {
+  
+ /* -- HEADER NAV -- */
+
+ .nav-bar {
+  padding-top: 1em;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 1.5em;
+}
+
+.nav-bar img {
+  width: 50px;
+}
+
+
+  nav ul {
+    justify-content: space-around;
+    column-gap: 2.25em;
+  }
+
+   /* -- CTA HEADER -- */
+ 
+  .cta-intro-inner {
+    row-gap: 1.5em;
+  }
+
+  .cta-intro-inner h1 {
+    font-size: 1.5rem;
+   }
+  
+  .cta-intro-inner h2 {
+   font-size: 1.2rem;
+  }
+
+  /* -- CTA BUTTON -- */
+
+  .cta-btn-header, .cta-btn {
+    font-size: 0.68rem;
```

Comment:
Kijk met zulke waardes uit dat je niet uitkomt op een decimaal qua pixels. Dit is bijvoorbeeld 10,88px. De browser is niet in staat om dat juist te doen

Reply:
Ah deze was mij ontglipt. Ik had bewust het gehele document doorgelopen en deze was ik vergeten. Deze nu ook aangepast. 

Via Netifly de landingspagina live gezet en ik kwam er daardoor achter dat de button style in het formulier door IOS wordt overruled. Deze samen met je opmerking aangepast door -webkit-appearance: none; te gebruiken. https://flexbox-landingpage.netlify.app/

------------------------------------------------------------------

Mag ik je nog het volgende vragen;

Zoals je kan lezen ben ik nogal uitgebreid met mijn toelichting en in mijn code zie ik dit ook terug. Is mijn gebruik van classes zoals ik deze toepas helder en niet teveel. Ik probeer voor mezelf te bepalen en aan te leren wat een goede werkwijze is. Wat zijn jouw ideeën hierover?

In deze opdracht heb ik bewust gekozen voor flexbox en dit zoveel mogelijk toegepast. Ik merk alleen dat je soms erg aan het zoeken bent waar de flexbox wordt gebruikt. Parent - child relatie. 
Is het niet soms makkelijker om de parent flexbox te maken en daarna de elementen de flow laten bepalen. Meer margin en padding toepassen. Groot voordeel wel dat ik bijna geen margin heb toegepast door het gebruik van Flexbox. Wat is jouw kijk hierop?

Heb jij nog interessante websites die mogelijkheden bieden om opdrachten uit te werken zoals bv deze opdracht. Dan kan ik nog extra oefenen om mijn CSS aan te scherpen.

Dit is niet de normale weg van communiceren. Ik hoor anders graag hoe ik dit wel het beste kan doen.

Alvast bedankt voor je reactie,

Met vriendelijke groet,

Michel van der Linden